layouts/community/ortho_4x10/default_ortho_4x10/keymap.c

```diff
@@ -0,0 +1,10 @@
+#include QMK_KEYBOARD_H
```

Comment:
Sorry that I didn't notice this before, but these should go in /layouts/default/ actually.